code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -217,17 +217,14 @@ GLOBAL_LIST_INIT(blacklisted_pylon_turfs, typecacheof(list(
 	var/corrupt_delay = 50
 	var/last_corrupt = 0
 
-/obj/structure/cult/functional/pylon/New()
+/obj/structure/cult/functional/pylon/Initialize(mapload)
 	. = ..()
+	START_PROCESSING(SSobj, src)
 	icon_state = SSticker.cultdat?.pylon_icon_state
 
 /obj/structure/cult/functional/pylon/attack_hand(mob/living/user)//override as it should not create anything
 	return
 
-/obj/structure/cult/functional/pylon/New()
```

Comment:
Duplicated `/New()` definition

Reply:
For the record, blood cult didn't touch these 😓